content/guides/translation-connection/step3-interact-with-julia.md

```diff
@@ -0,0 +1,241 @@
+  * Translation submitter
+  * Translation queued
+  * Translation locales
+  * Translation due date
+4. Now set the translation attributes:
+  * `Translation name`: enter the desired name for the translation project.
+  * `Translations submitter`: select your name (Julia's name).
+  * `Translation queued`: enable the attribute by sliding the toggle button towards `Yes`.
+  * `Translation locales`: select the desired target locales for translation.
+  * `Translations due date`: Select the date on which you want to receive the translation.
+5. Click on `NEXT` and `CONFIRM` the operation.
+
+After this last action, the `Bulk action` will run and Julia will be notified in the Akeneo PIM notification bar once it is successfully executed.
+
+With the attributes of the products Julia wants to have translated now ready, your connector is ready to go!
+
+### Translation process
+
+At regular intervals, your connector will need to retrieve products where `Translation queued` attribute has been set to Enable (set to `true`).
+
+You can do this by using our PIM [API filtering system](https://api.akeneo.com/documentation/filter.html#filter-on-product-values) on product attribute value.
+
+Then your connector need to set this attribute to `false` and change the `Translation status` attribute to `IN PROGRESS` to indicate to Julia that the translation is being processed by one of your translator.
+
+:::tips
+Using the previous [GET LIST OF PRODUCTS](https://api.akeneo.com/api-reference.html#get_products) API request (with the correct filter), your connector will retrieve a JSON structure of each localizable products.
+
+Then with the help of the `family code` of these products, your connector will be able to make a [GET LIST OF ATTRIBUTES](https://api.akeneo.com/api-reference.html#get_attributes) API request to retrieve the `type` of attributes that compose this product family.
+
+By analyzing these types, you will be able to define for your translator which are the textual attributes he needs to translate.
+:::
+
+When the translation is finished, your translator need to set the `Translation status` attribute to `TRANSLATED` and import all translated product data back to the PIM.
+
+If for some reasons the translation project has been cancelled, your translator need to set the `Translation status`attribute to `CANCELLED` and import this product data back to the PIM.
+
+:::info
+To develop theses connector actions, and to understand our API please follow our [wonderful dedicated API documentation](https://api.akeneo.com)!
+:::
+
+### Check Translation status
+
+Now let's go back to Julia's side and see how she can check her translation projects status.
+
+As the `Translation status` has been declared as `Usable in grid`, Julia can:
+* [Customise her grid](https://help.akeneo.com/pim/serenity/articles/products-grid.html#customise-the-grid) to add the `Translation status`: this will allow her to sort his products according to their translation status.
```

Comment:
```suggestion
* [Customise her grid](https://help.akeneo.com/pim/serenity/articles/products-grid.html#customise-the-grid) to add the `Translation status`: this will allow her to sort her products according to their translation status.
```